src/otx/algo/segmentation/dino_v2_seg.py

```diff
@@ -68,3 +70,46 @@ def to(self, *args, **kwargs) -> Self:
             msg = f"{type(self).__name__} doesn't support XPU."
             raise RuntimeError(msg)
         return ret
+
+
+class DinoV2SegSemiSL(OTXDinoV2Seg):
+    """DinoV2SegSemiSL Model."""
+
+    def _customize_inputs(self, entity: SegBatchDataEntity) -> dict[str, Any]:
+        if not isinstance(entity, dict):
+            if self.training:
+                msg = "unlabeled inputs should be provided for semi-sl training"
+                raise RuntimeError(msg)
+            return super()._customize_inputs(entity)
+
+        entity["labeled"].masks = torch.stack(entity["labeled"].masks).long()
+        w_u_images = entity["weak_transforms"].images
+        s_u_images = entity["strong_transforms"].images
+        unlabeled_img_metas = entity["weak_transforms"].imgs_info
+        labeled_inputs = entity["labeled"]
+
+        return {
+            "inputs": labeled_inputs.images,
+            "unlabeled_weak_images": w_u_images,
+            "unlabeled_strong_images": s_u_images,
+            "global_step": self.trainer.global_step,
+            "steps_per_epoch": self.trainer.num_training_batches,
+            "img_metas": labeled_inputs.imgs_info,
+            "unlabeled_img_metas": unlabeled_img_metas,
+            "masks": labeled_inputs.masks,
+            "mode": "loss",
+        }
+
+    def _create_model(self) -> nn.Module:
+        # merge configurations with defaults overriding them
+        backbone_configuration = DinoV2Seg.default_backbone_configuration | self.backbone_configuration
+        decode_head_configuration = DinoV2Seg.default_decode_head_configuration | self.decode_head_configuration
+        backbone = DinoVisionTransformer(**backbone_configuration)
+        decode_head = FCNHead(num_classes=self.num_classes, **decode_head_configuration)
+        base_model = DinoV2Seg(
+            backbone=backbone,
+            decode_head=decode_head,
+            criterion_configuration=self.criterion_configuration,
+        )
+
+        return MeanTeacher(base_model, unsup_weight=0.7, drop_unrel_pixels_percent=20, semisl_start_epoch=2)
```

Comment:
`unsup_weight`, `drop_unrel_pixels_percent`, `semisl_start_epoch` - Do we need to make these values configurable?

Reply:
Yeah, I thought about that, but didn't come up the final solution with that. Could we change this in my following PR with Segmentation task refactoring?